src/expr/src/scalar/func/macros.rs

```diff
@@ -142,7 +141,7 @@ macro_rules! sqlfunc {
                     }
                 }
             }
-            pub use $fn_name::[<$fn_name:camel>];
+            pub use [<__ $fn_name _impl>]::[<$fn_name:camel>];
```

Comment:
To give me a small headstart on making sense of this, where does this syntax come from? `paste`?

Reply:
Yup! https://docs.rs/paste/1.0.5/paste/#pasting-identifiers